test/test_transforms_v2_refactored.py

```diff
@@ -2280,3 +2281,59 @@ def resize_my_datapoint():
         _register_kernel_internal(F.resize, MyDatapoint, datapoint_wrapper=False)(resize_my_datapoint)
 
         assert _get_kernel(F.resize, MyDatapoint) is resize_my_datapoint
+
+
+class TestPermuteChannels:
+    _CORRECTNESS_PERMUTATIONS = list(itertools.permutations(range(3)))
+    _DEFAULT_PERMUTATION = _CORRECTNESS_PERMUTATIONS[-1]
```

Comment:
Please get rid of that and hard-code its actual value where it's used. It's impossible to know what's being permuted otherwise.

That also means we can get rid of `_CORRECTNESS_PERMUTATIONS`.